modules/text_server_adv/text_server_adv.cpp

```diff
@@ -2112,10 +2112,10 @@ Dictionary TextServerAdvanced::_font_get_ot_name_strings(const RID &p_font_rid)
 				name = vformat("unknown_%d", names[i].name_id);
 			} break;
 		}
-		String text;
 		unsigned int text_size = hb_ot_name_get_utf32(hb_face, names[i].name_id, names[i].language, nullptr, nullptr) + 1;
-		text.resize(text_size);
-		hb_ot_name_get_utf32(hb_face, names[i].name_id, names[i].language, &text_size, (uint32_t *)text.ptrw());
+		char32_t *buffer = memnew_arr(char32_t, text_size);
```

Comment:
In this case it's not a big deal, but we probably should add resize to the godot-cpp instead to avoid buffer copying. Also, who's freeing the buffer?

Reply:
I agree!

> Also, who's freeing the buffer?

This PR is... now :-) I had to trace through the code a bit, but it looks like it's only copied from